### ISSUE TITLE
Using Two Instances Of CSS

### ISSUE THREAD

**Issue by zsheley** [zsheley/photos#1]
I am trying to use two sections of CSS. The current part in the index.html file and the instance in the index.css file. However, when I try to link the external file it only uses that CSS instead of in union with the CSS included in index.html/

**Comment by zsheley** [comment]
If you have any questions about the issue feel free to ask.

**Comment by NikithaNadig** [comment]
Can I work on this?

**Comment by zsheley** [comment]
Yeah sure. As far as the CSS goes. Otherwise just ask before editing anything else

**Comment by zsheley** [comment]
Thank you!

**Comment by NikithaNadig** [comment]
I have made some changes. Have a look at it

**Comment by zsheley** [comment]
Yes that works great! When I add pictures will they automatically organize into rows?
